webextensions/api/extensionTypes.json

```diff
@@ -17,10 +17,12 @@
                 "notes": "This feature is supported but not exposed through the 'extensionTypes' object."
```

Comment:
This existing comment also applies to Firefox. There is no `extensionTypes.ImageDetails` in the API interface.